apps/api/prisma/schema.d.ts

```diff
@@ -187,23 +189,45 @@ export type CaseStatusNameType =
 	| 'Withdrawn';
 
 export interface DocumentDetails {
-	guid: string;
+	documentId: string | null;
+	sourceSystem: string;
+	documentGuid: string | null;
+	redacted: boolean;
```

Comment:
this is duplicated

Reply:
also need some updates form this morning's chat in Teams - updated list should be available soon https://pins-ds.atlassian.net/wiki/spaces/BO/pages/1064730651/Document+Metadata+-+DRAFT

Reply:
@francescobrenci-AR whats duplicate